package-lock.json

```diff
@@ -794,7 +794,7 @@
     },
     "ansi-escapes": {
       "version": "3.1.0",
-      "resolved": "https://registry.npmjs.org/ansi-escapes/-/ansi-escapes-3.1.0.tgz",
+      "resolved": "http://registry.npmjs.org/ansi-escapes/-/ansi-escapes-3.1.0.tgz",
```

Comment:
Just a bit strange that it changed from https to http. Is there a reason for that?

Reply:
Not sure. Didn't make this change manually - `npm install` did. I'm on version `6.14.12`